src/client/components/Meals.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import DisplayMeal from "./DisplayMeal";
+function Meals() {
+  const [input, setInput] = useState([]);
+  const [mealItems, setMealItems] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (input === "") {
```

Comment:
Here you have duplication with the fetching logic. Think about how you can create something generic to handle to fetching